src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    protected String color;
+
+    public abstract String toString();

```

Comment:
According to the task description, you should not use the `toString()` method for 'drawing' figures. Instead, create a separate method to print out all information about the figure using `System.out.println()`.